fabric/patches/2_0004-Deploy-qscc-via-opt_scc-extension-point.patch

```diff
@@ -0,0 +1,86 @@
+From 0505d4c4c4f7ac665afced25975840be299fe9dc Mon Sep 17 00:00:00 2001
+From: michael steiner <michael.steiner@intel.com>
+Date: Wed, 19 Aug 2020 15:40:39 -0700
+Subject: [PATCH 1/4] Deploy qscc via opt_scc extension point
+
+Signed-off-by: michael steiner <michael.steiner@intel.com>
+---
+ core/scc/opt_scc/qscc.go    | 28 ++++++++++++++++++++++++++++
+ internal/peer/node/start.go |  5 +----
+ 2 files changed, 29 insertions(+), 4 deletions(-)
+ create mode 100644 core/scc/opt_scc/qscc.go
+
+diff --git a/core/scc/opt_scc/qscc.go b/core/scc/opt_scc/qscc.go
+new file mode 100644
+index 000000000..334e75494
+--- /dev/null
++++ b/core/scc/opt_scc/qscc.go
+@@ -0,0 +1,28 @@
++//+build !test
```

Comment:
seems weird to me. only build when not testing? what gets built when testing? The upstream qscc?

I may be missing some context, but maybe it could be solved by a comment with some of the intent described.

Reply:
yes, this is only needed to exclude qscc which you probably want to unconditionally build but for all the test cases to work you have to exclude qscc's init during testing (via `go test --tags test`).  I would guess the qscc patch will not be upstreamed but still thought it would be a good demonstration on how the framework would be used.   If it is not upstreamed, then you could also drop the `--tags test` from the test command ...